stream-chat-android-offline/src/main/java/io/getstream/chat/android/offline/extensions/ChatClient.kt

```diff
@@ -303,3 +305,15 @@ internal fun ChatClient.shuffleGiphy(message: Message): Call<Message> {
         }
     }
 }
+
+/**
+ * Deletes the channel with the specified id.
+ *
+ * @param cid The full channel id i. e. messaging:123.
+ *
+ * @return Executable async [Call] deleting a channel.
+ */
+@CheckResult
+public fun ChatClient.deleteChannel(cid: String): Call<Unit> = CoroutineCall(domainImpl().scope) {
```

Comment:
Why do we need this extension? I think we should suggest using `ChatClient.instance().channel(cid).delete()` or `ChatClient.instance().delete(channelType, channelId)` and skip it. WDYT?

Reply:
Done